packages/core/contracts/oracle/implementation/SkinnyOptimisticOracle.sol

```diff
@@ -612,9 +691,19 @@ contract SkinnyOptimisticOracle is SkinnyOptimisticOracleInterface, Testable, Lo
         return request.bond.div(2);
     }
 
-    function _validateLiveness(uint256 liveness) private pure {
```

Comment:
This method is only used in constructor and we can't use it in other methods that take in `customLiveness` because we allow `customLiveness` to be equal to 0.